Security/src/Steeltoe.Samples.Security.sln

```diff
@@ -0,0 +1,71 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.10.34707.107
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RedisDataProtectionKeyStore", "RedisDataProtectionKeyStore\RedisDataProtectionKeyStore.csproj", "{3393201D-0A45-4812-9888-5BFB89F5F76D}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{E89A1BB6-1F81-4C2A-9F4A-936EE33ACD8B}"
+	ProjectSection(SolutionItems) = preProject
+		..\..\CommonTasks.md = ..\..\CommonTasks.md
+		..\..\configure.ps1 = ..\..\configure.ps1
```

Comment:
I'm leaning toward deleting this script entirely (or at least removing the reference here).

The reason to delete is that samples typically are updated at different times. Running the script potentially breaks other samples and we don't want to update/retest everything all the time. Furthermore, the template at `/config/Directory.Build.props` is not only outdated, but what's worse is it contains just a small subset of "common" versions. So updating the file without also updating all the explicit sample-specific NuGet references likely breaks most samples.

Reply:
Yeah, at this point we're so out of sync it probably makes sense to scrap the script